### DISCUSSION TITLE
How to use fast prompts?

### DISCUSSION THREAD

**Discussion by conderus** [lencx/Noi#169] [Q&A]
Hi, I was using other App before, and I could directly execute prompts in chatgpt chat, with shortcut /, for example “/poet” was assigned to particular prompt. It was auto searched and executed when I hit enter. Is it possible to use it this way here? Because now I must search prompt, copy and paste to chatgpt. I can do the same with notepad and I don’t need this build-in database with prompts, tags etc.

**Comment by SorryFB** [comment]
I need to know also.

**Comment by PawkyFox** [comment]
Yes, dear @lencx I also don`t understand. How can I apply inserting selected prompt into previously active window: in the Noi app, or at any another previously active app, as it works in DeepL translator

You just pressed keys combination, in the appeared window you see the translation and just next button — Boom! That text will be inserted into the previously active input field whatever app it was. Mostly it pastes it where you`ve take it..
Or that doesn`t intended to be like that?

Asking with a great respect! The app itself is great! Just start to use it, but unsure if I get it completely

**Reply by kerzhaev** [reply]
app is not update for a long time. Recently this feature work well and now does not work.
